Makefile

```diff
@@ -61,6 +61,9 @@ endif
 test-examples:
 	$(REBAR3) eunit --dir=examples_test
 
+parallel-test-examples:
```

Comment:
Rename this to `test-parallel` (for uniformity).
Also, add it to the appropriate place in the `.PHONY:` line.